README.md

```diff
@@ -23,7 +22,7 @@ Additional description about the project and its features.
 ## Live Demo
 
 You can check the link below for the live demo.
-N/A
+https://ewoskyx.github.io/leaderboard/
 
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Hello, @rloterh! Thank you for the review. I will change it immediately.